CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+
+#### Set up `mdn/content` repo
+
+First, you will need to set up the `mdn/content` repo locally. If you do not plan to contribute to upstream content, you will not need to fork it as described in its [Contributing guide](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#forking-and-cloning-the-repository); all you need to do is clone it by running the following command:
+
+```bash
+git clone git@github.com:mdn/content.git
+```
+
+After the repository has been cloned, follow the [steps to prepare the project](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#preparing-the-project).
+
+#### Set up `mdn/translated-content` repo
+
+Once that is done, you will also need to fork and clone this repository by following the [upstream Contributing guide](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#forking-and-cloning-the-repository), replacing `content` with `translated-content`.
+
+#### Link translated content
+
+1. Navigate to your local clone of the content repository fork:
+
+   ```bash
+   cd /path/to/content
+   ```
+
+2. Next, define an environment variable in a `.env` file called `CONTENT_TRANSLATED_ROOT` containing the path to the _translated-content_ repo’s `files` directory:
+
+   ```bash
+   # macOS/Linux
+   echo CONTENT_TRANSLATED_ROOT=/path/to/translated-content/files >> .env
+   ```
+
+   ```batch
+   # Windows
+   echo CONTENT_TRANSLATED_ROOT=\path\to\translated-content\files > .env
+   ```
+
+   (the `.env` file will be created for you if it does not already exist.)
+
+3. Run the command `yarn start` to start the local server at `localhost:5042`.
+
+#### Working in the translated-content repo
+
+This repo has exactly the same folder structure and concepts as the [upstream content repo](https://github.com/mdn/content). The main difference is in the setup you need to do before you can start editing. It is mostly the same, but there is a little bit more to consider. Primarily, commands such as `yarn content` are only available from the upstream content repo; linting commands are available here, however, as they often use different configuration.
+
+## Contributing to MDN translated content
+
+This section describes how to perform the most common types of contributions to MDN translated content.
+
+### Fix a typo
+
+To fix a typo, refer to the [Simple changes](#simple-changes) section.
+
+### Create a new translation
+
+To create a new translation for a page that does not yet have one, perform the following steps. For this example, we'll assume you are trying to translate the MDN landing page into French.
+
+1. Find the source file in the upstream content repository. (ex. `files/en-us/mdn/index.md`)
+2. Copy the file to the appropriate locale's folder. (ex. `cp files/en-us/mdn/index.md files/fr/mdn/index.md`)
```

Comment:
Emmmm, the directory to content repo would not be the same as the translated-content root. Should we specific the path to upstream document like this?

```suggestion
1. Find the source file in the upstream content repository. (ex. `/path/to/content/files/en-us/mdn/index.md`)
2. Copy the file to the appropriate locale's folder. (ex. `cp /path/to/content/files/en-us/mdn/index.md files/fr/mdn/index.md`)
```